tuf/src/repo_builder.rs

```diff
@@ -858,22 +858,32 @@ where
     }
 
     fn need_new_targets(&self) -> bool {
-        // If we don't have a database yet, we need to stage the first targets metadata.
+        // We need a new targets if we added and targets.
```

Comment:
I think you mean any targets

Reply:
Oops, fixed this and some other bad comments too.